### PR TITLE
comments for a single article view added with cool accordion style

### PR DESCRIPTION
**Pull request by safchik**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                        {comments.map((comment) => (
+                            <li key={comment.comment_id}>
+                                <p>{comment.body}</p>
+                                <p>By: {comment.author}</p>
+                            </li>
+                        ))}
+                    </ul>
+                )}
+            </div>
+        </section>
```
Comment (itskatherine) [inline src/components/Comments.jsx:43]: Cool use of an accordion design!
However in some cases (for example in article 34 for me) the bottom comment is being cropped out of view - could you take a look at your css to make sure you can see all the comments 🙂

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    return (
+        <section id="comments">
+            <button className="accordion" onClick={handleToggleClick}>
+                {isExpanded ? "Hide comments" : "Show comments"}
+            </button>
+            <div className={`panel ${isExpanded ? "expanded" : ""}`}>
+                {isLoading ? (
+                    <h3>Loading...</h3>
+                ) : comments.length === 0 ? (
+                    <p>No comments to display.</p>
```
Comment (itskatherine) [inline src/components/Comments.jsx:30]: Great that you've accounted for when there are no comments

File: src/components/Comments.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                ) : (
+                    <ul>
+                        {comments.map((comment) => (
+                            <li key={comment.comment_id}>
```
Comment (itskatherine) [inline src/components/Comments.jsx:34]: You can get rid of bullet points in list elements by using the `list-style-type: none` property in css

File: src/components/Comments.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                                <p>{comment.body}</p>
+                                <p>By: {comment.author}</p>
```
Comment (itskatherine) [inline src/components/Comments.jsx:36]: I think ideally the comments would also include when they were written (i.e. the time/date they were created_at)

**Review by itskatherine — COMMENTED** [review]
Hi Dmytro, nice work 👍 ! Have a look at my comments and once you've address them, feel free to merge

File: src/components/Comments.jsx (hunk 4 above)
Comment (safchik) [reply src/components/Comments.jsx:36]: sorted

File: src/components/Comments.jsx (hunk 3 above)
Comment (safchik) [reply src/components/Comments.jsx:34]: sorted

**Pull request merged**